x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/form.tsx

```diff
@@ -373,9 +377,11 @@ export function useOnSubmit({
       } else {
         if (isVerificationError(error)) {
           setFormState('VALID'); // don't show the add agent modal
-          const forceInstall = await confirmForceInstall(packagePolicy.package!);
+          const forceInstallUnverifiedIntegration = await confirmForceInstall(
+            packagePolicy.package!
```

Comment:
wouldn't it be safer to add the `packagePolicy.package` to the if statement instead of using non null assertion?

Reply:
in this line only the variable name changed, the rest is the same as it was. As we are not the code owner, I wouldn't touch what is not broken :)